index.html

```diff
@@ -868,7 +952,7 @@ <h3>Controllers</h3>
 While the identifier used for a [=controller=] is unambiguous, this does not
 imply that a single entity is always the controller, nor that a controller
 only has a single identifier. A [=controller=] might be a single entity, or a
```

Comment:
```suggestion
only has a single identifier. A [=documentController=] might be a single entity, or a
```

Reply:
-1 for the reasons stated here: https://github.com/w3c/controller-document/pull/116/files#r1839014775

Reply:
I'm marking all of @David-Chadwick's document controller / proof controller suggestions as resolved as they are now in a new issue #119